app/views/content/urgent-call-for-qualified-teachers/_faq.html.erb

```diff
@@ -330,12 +330,13 @@
     </p>
 
     <p>
-      If you’re thinking about returning to teach modern foreign languages, maths or physics
-      at secondary school in England, you can <a href="/tta-service">get a return to teaching adviser</a> who can give
-      you free one-to-one support on applying for roles, writing personal statements and
-      preparing for interviews. They can also help you find courses to enhance your subject
-      knowledge.
-    </p>
+      If you’re thinking about returning to teach at secondary school in England, you could
+      be eligible for a <a href="/tta-service">return to teaching adviser</a>. You’ll need
+      to be qualified to teach modern foreign languages, maths, chemistry, computing
```

Comment:
I think this needs rewording slightly - more about wanting to teach MFL, maths etc. and have qualified teacher status (QTS), rather than be qualified to teach MFL or maths etc.?